core/bin/migrate_database

```diff
@@ -3,6 +3,11 @@
 
 import startup  # noqa: autoflake
 
+from core.config import Configuration
+from core.model import SessionManager
 from core.scripts import AlembicMigrateVersion
 
-AlembicMigrateVersion().run()
+session = SessionManager.session(
+    Configuration.database_url(), initialize_data=False, initialize_schema=False
```

Comment:
Might be helpful to add a comment here with some context as to why we need to set `initialize_data` and `initialize_schema` to `False`.